src/axolotl/utils/gradient_checkpointing/offload_disk.py

```diff
@@ -0,0 +1,531 @@
+"""
+DISCO - DIsk-based Storage and Checkpointing with Optimized prefetching
```

Comment:
Is this your name for this method? A quick google search doesn't turn anything up